model/project_ref.go

```diff
@@ -2434,6 +2434,44 @@ func GetMessageForPatch(patchID string) (string, error) {
 	return project.CommitQueue.Message, nil
 }
 
+func ValidateContainers(project string, containers []Container) error {
+	pRef, err := FindMergedProjectRef(project, "", true)
+	if err != nil {
+		return errors.Errorf("Unable to find merged project ref for project %s", project)
```

Comment:
There also needs to be a nil pRef check after this since `FindOne`s typically `return nil, nil` when the document is not found.